pkg/repositories/repositories.go

```diff
@@ -58,6 +58,41 @@ func NewConfig(filename string) (Config, error) {
 	return repoConfig, nil
 }
 
+// UpdateConfigVersions updates a config in-place, setting the afterVersion to
+// its version, and it's version to a newer one specified in a secondary config
+func UpdateConfigVersions(baseConfig *Config, configWithNewVersions *Config) {
```

Comment:
This is more of a personal style preference, but the *Config params could be consolidated:
```
func UpdateConfigVersions(baseConfig, configWithNewVersions *Config) {
```

Reply:
I think for now its better if we keep the type for each param, since we don't use that style pattern anywhere else in this package.